src/PerfView/App.cs

```diff
@@ -788,6 +788,12 @@ public static string SourcePath
         /// </summary>
         public static SymbolReader GetSymbolReader(string etlFilePath = null, SymbolReaderOptions symbolFlags = SymbolReaderOptions.None)
         {
+            // Apply the DisableNGenPdbGeneration option if it's set in the command line
```

Comment:
Looking a bit further at the code, there is already a `/NoNGenPdbs` flag that disables NGEN PDB generation.  Please undo this entire change.

Instead, please change this PR to remove .NET Core PDB generation that uses crossgen.  This is implemented in SymbolReader.cs.  The call to HandleNetCorePdbs is what does this.  The entire function should be removed.  You can also remove `GetCrossGenExePath`.